examples/android/app/build.gradle

```diff
@@ -0,0 +1,219 @@
+apply plugin: "com.android.application"
```

Comment:
this file comes from https://github.com/facebook/react-native/blob/v0.63.4/template/android/app/build.gradle